Signal/src/ViewControllers/ContactAccount.h

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class Contact;
+
+// We want to be able to present contacts with multiple signal
+// accounts in the UI.  This class represents a given (contact,
+// signal account) tuple.
+@interface ContactAccount : NSObject
+
+@property (nonatomic) Contact *contact;
+
+// An E164 value identifying the signal account.
+@property (nonatomic) NSString *recipientId;
+
+//// TODO:
+//@property (nonatomic) NSString *displayName;
```

Comment:
I'm leaving this class a bit incomplete since I'm going to revisit it almost immediately.  See above.